coderd/database/migrations/000126_login_type_none.up.sql

```diff
@@ -1,3 +1,34 @@
-ALTER TYPE login_type ADD VALUE IF NOT EXISTS 'none';
+-- This migration has been modified after its initial commit.
+-- The original disregarded the warning in create_migration.sh.
```

Comment:
Nit: I don't think we can explicitly say the warning was disregarded; that implies volition, and it was probably just done this way because, you know, it's the way to do it 🙂

Reply:
True. Let's blame the missed connotation on English being my second language. No offence meant to to original author. Will reword.

Reply:
I figured as much :) I'm sure no offence was taken.

Reply:
Just to make sure:

@Emyrk Are we good? I was only trying to document for archaeology's sake. Apologies for inadvertent offence. I should have found more neutral language first time.